docs/testnet/essentials.md

```diff
@@ -1,22 +1,20 @@
-# Essential Testnet Information
-An easy-to-read list of essential parameters and configuration settings for Obscuro Testnet.
+# Essential Information
+Essential information, parameters and configuration settings for the Oikos testnet.
+
+## Limitations on Oikos
+These are the key limitations to be aware of when developing for the Oikos testnet:
```

Comment:
A lot of these limitations wouldn't be limitations related to _developing_ for testnet. Maybe just say, "key limitations to be aware of when using the Oikos testnet".